client/src/components/relatedProducts/ProductCard.jsx

```diff
@@ -1,12 +1,11 @@
 import React, {useState, useEffect} from 'react';
 import EachCard from './EachCard.jsx';
+import YourOutfitAdder from './YourOutfitAdder.jsx';
 
 const ProductCard = (props) => {
   if (!props.detailInfo) {
     return null;
   }
-
-
   let combine = props.detailInfo.reduce((map, value) => {
     map[value.product_id] = value;
     return map;
```

Comment:
Just curious for myself about this syntax... could you return map by saying `return map[value.product_id] = value;` all in one line? Or is it different?

Reply:
You could, but in this case I want to assign the returned value of the reduce function to the variable combine because then I'm manipulating the object that is assigned to combine.